docs/source/contents/4_draft_mapping.rst

```diff
@@ -0,0 +1,189 @@
+or one of the CC licenses is present, the value should be mapped to edm:rights and have a value type of URI.
+
+**Justification**
+
+DPLA maps both CC licenses and Rights Statements to edm:rights. So does Samvera. Presently only the heilman collection includes
+a CC license
+
+**Xpath**
+
+mods:accessCondition[@xlink:href]
+
+**Decision**
+
+`Example record for Rights Statements <https://digital.lib.utk.edu/collections/islandora/object/knoxgardens%3A115/datastream/MODS>`_
+
+.. code-block:: xml
+
+    <accessCondition type="use and reproduction"
+                    xlink:href="http://rightsstatements.org/vocab/CNE/1.0/">
+        Copyright Not Evaluated
+    </accessCondition>
+
+.. code-block:: turtle
+
+    @prefix edm: <http://www.europeana.eu/schemas/edm/> .
+
+
+        <https://example.org/objects/1> edm:rights <http://rightsstatements.org/vocab/CNE/1.0/> .
+
+
+`Example record for CC license <https://digital.lib.utk.edu/collections/islandora/object/heilman%3A1010/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <accessCondition type="use and reproduction"
+                    xlink:href="https://creativecommons.org/licenses/by-nc-nd/3.0/">
+        Attribution-NonCommercial-NoDerivs 3.0 Unported (CC BY-NC-ND 3.0)
+    </accessCondition>
+
+.. code-block:: turtle
+
+    @prefix edm: <http://www.europeana.eu/schemas/edm/> .
+
+
+        <https://example.org/objects/1> edm:rights <https://creativecommons.org/licenses/by-nc-nd/3.0/> .
+
+accessCondition - Restrictions on Access
+----------------------------------------
+
+**Use case**
+
+The Howard Baker Audiovisual Collection includes 46 items that are "In Copyright" and therefore have restricted access to
+avoid any potential copyright conflicts. Only on campus access is provided to the actual recordings, though the metadata
+records are accessible from anywhere. Having the metadata accessible makes it so that anyone can discover these materials
+and decide for themselves if it is worth a trip into campus. Some of the recordings had some deterioration and were therefore
+digitized as a preservation measure. Having digitized copies also made providing on site access easier. In order to make
+sure that users are aware of the on campus only restriction, a note needed to be added to the metadata. When off campus
+users visit the metadata, this note makes it clear why they cannot access the recording.
+
+**Justification**
+
+As the value present in the current accessCondition node is not associated with a controlled vocabulary and simply needs to
+be displayed to the user within the record, there is no reason to connect it with other accessCondition values. A note is
+sufficient for this use case.
+
+**Xpath**
+
+mods:accessCondition[@type="restriction on access"]
+
+**Decision**
+
+`Example record <https://digital.lib.utk.edu/collections/islandora/object/bakerav%3A10/datastream/MODS/view>`_
```

Comment:
@mlhale7, we'll talk about this later in greater detail, but to get that dumb warning to go away, you need to change your syntax to this:

```

`Example record <https://digital.lib.utk.edu/collections/islandora/object/bakerav%3A10/datastream/MODS/view>`__

```

Reply:
@markpbaggett - sorry, I didn't have Pygments appropriately installed. It looked like the links needed a distinct textual reference so I just changed my standard. Hopefully this fixes things.